PowerShellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingServiceUtilities.cs

```diff
@@ -403,7 +403,9 @@ private void EnterCredentialedRemoteSession(PowerShell powerShell, string remote
                 return;
             }
 
-            string port = remoteName.Split(':').ElementAtOrDefault(1);
+            string[] addressParts = remoteName.Split(':');

```

Comment:
Also, will this handle it correctly if user input something other than an IP address?